frontend/src/index.html

```diff
@@ -40,9 +40,6 @@
   <!-- required for lhncbc/lforms -->
   <script src="./assets/js/webcomponents/webcomponents-loader.js"></script>
 
-  <!-- wails -- this file is expected to be missing in docker/web mode. It's only required in Desktop -->
-  <script type="module" src="/wails/runtime.js"></script>
-
```

Comment:
can you revert the deletion of this section?

Reply:
I'll probably refactor this so that it's not attempted in docker mode.